### ISSUE TITLE
Light mode suddenly got real dull

### ISSUE THREAD

**Issue by whippingdot** [ceciliamay/obsidianmd-theme-primary#84]
I have transparency on, but still light mode never seemed this dull (I just tested it out cause of the new 1.13.1 and i wanted to see the new 'light' cursor)
![image](https://user-images.githubusercontent.com/64354499/156936064-b3ff3d9d-e624-4ca8-82b7-910886ffdbe5.png)

Btw light cursor basically is the active line being lighter instead of darker.

**Comment by whippingdot** [comment]
Oh I just turned off transparency. It is because of transparency...

I think transparency might be broken for light mode? Or maybe this is how it always was. I feel like I would've realized tho cause i have been using this theme for quiet some while now.

**Comment by whippingdot** [comment]
Sorry for not creating this in bugs - I never noticed bugs was a new tab

**Comment by ceciliamay** [comment]
Hey there! Yeah, I currently don't support transparency mode because of my design vision for Primary... I currently have no ideas, but I might look into it. So for now my solution would be not to use transparent mode. 👀

Could you explain to me a bit about what you like about transparency mode, why you like it? I don't personally use it, so I'd love to hear from a user - such as you - that do!

Also, don't worry about the issue labels - I file them ;)

**Comment by whippingdot** [comment]
I like the transparency mode because I feel like it fits in with windows 11 (which i am using right now). I like the non-solidness of the background, and it is really cool being able to see background applications, though kinda blurry.

**Comment by ceciliamay** [comment]
Hey there @whippingdot !! So I looked into this. And I improved on it a little bit to make it match Win 11 a bit more (sidebars are more translucent than note). However, Dark Mode really looks better with transparency mode (as noted by Obsidian):
![image](https://user-images.githubusercontent.com/90643719/161005720-9c7776e3-e064-4651-8d31-397303f1880a.png)
vs. Light Mode
![image](https://user-images.githubusercontent.com/90643719/161005881-ca213ac7-4077-4f0a-b8f3-4c78ee3f9e14.png)

I tried different settings but this is the best I can do (for now). I don't know if it has achieved the effect you are looking for but I will try to look into this in the future. I will close this for now, but put it on the roadmap for future experiments :) 

Thank you so much for this suggestion+report!! 🫂
